### ISSUE TITLE
Carthage build fail in release 1.0.1

### ISSUE THREAD

**Issue by bidadh** [RxSwiftCommunity/RxGesture#47]
I'm getting the following error with your latest release(1.0.1) using Carthage.
it was working before so I have to explicitly specify 1.0.0 in my Cartfile which is not my preferred way.

**Building scheme "RxGesture-iOS" in RxGesture.xcodeproj
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project PATH_AO_APP/Carthage/Checkouts/RxGesture/RxGesture/RxGesture.xcodeproj -scheme RxGesture-iOS -configuration Release -derivedDataPath ../Caches/org.carthage.CarthageKit/DerivedData/RxGesture/1.0.1 -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build**

I tried to build it and the error is **PATH_TO_APP/Carthage/Checkouts/RxGesture/Pod/Classes/View+RxGesture.swift:102:22: error: value of type 'Observable<G>' has no member 'bind'
                    .startWith(gesture)**

FYI:
I had a look into the file and compared with the previous version

```swift
                let disposable = genericGesture.rx.event
                    .map { $0 as! G }
                    .startWith(gesture)
                    .bindNext(observer.onNext)
```

has been changed to:
```swift
                let disposable = genericGesture.rx.event
                    .map { $0 as! G }
                    .startWith(gesture)
                    .bind(onNext: observer.onNext)
```
rollback works so if I change this line build succeeds

**Comment by jegnux** [comment]
@bidadh I just ran `carthage update --platform iOS` on a fresh project with the following Cartfile and it went well.

```swift
github "ReactiveX/RxSwift"
github "RxSwiftCommunity/RxGesture"
```

What's your cartfile content?
Also, I'm not sur to understand, what it the code that works? `.bind(onNext:` or `.bindNext(`?
#45 changed `.bindNext(` to `.bind(onNext:` because the former was deprecated by RxSwift 3.4.0.

**Comment by bidadh** [comment]
here is my Cartfile
```
github "Alamofire/Alamofire"
github "ReactiveX/RxSwift"
github "RxSwiftCommunity/RxAlamofire"
github "RxSwiftCommunity/RxOptional"
github "RxSwiftCommunity/RxGesture"
github "RxSwiftCommunity/Action"
github "RxSwiftCommunity/RxSegue"
github "RxSwiftCommunity/RxSwiftExt"
github "RxSwiftCommunity/RxDataSources"
github "devxoul/RxKeyboard"
github "Swinject/Swinject"
github "Swinject/SwinjectStoryboard"
github "adamwaite/Validator"
github "jasonnam/Navajo-Swift"
github "Hearst-DD/ObjectMapper"
github "MengTo/Spring" "swift3"
```

And the answer to your second question is `.bindNext` works

**Comment by jegnux** [comment]
@bidadh If `.bind(onNext:` doesn't work, it means that you don't have `RxSwift 3.4.0` but an older release. 
What is in your `Cartfile.resolved`?

**Comment by bidadh** [comment]
it's 3.4.0

**Comment by bidadh** [comment]
here is my ` Cartfile.resolved`

```
github "Alamofire/Alamofire" "4.4.0"
github "Hearst-DD/ObjectMapper" "2.2.5"
github "MengTo/Spring" "8201b5c93bd7ebdf4cd18160bcde1e6e88c2be64"
github "Quick/Nimble" "v5.1.1"
github "Quick/Quick" "v1.1.0"
github "ReactiveX/RxSwift" "3.4.0"
github "RxSwiftCommunity/Action" "2.2.2"
github "RxSwiftCommunity/RxAlamofire" "3.0.2"
github "RxSwiftCommunity/RxDataSources" "1.0.3"
github "RxSwiftCommunity/RxGesture" "1.0.0"
github "RxSwiftCommunity/RxOptional" "3.1.3"
github "RxSwiftCommunity/RxSegue" "1.0.0"
github "RxSwiftCommunity/RxSwiftExt" "2.3.0"
github "Swinject/Swinject" "2.0.0"
github "Swinject/SwinjectStoryboard" "1.0.0"
github "adamwaite/Validator" "v2.1.1"
github "devxoul/RxKeyboard" "0.4.1"
github "jasonnam/Navajo-Swift" "1.0.0"
```

**Comment by jegnux** [comment]
```swift
github "RxSwiftCommunity/RxGesture" "1.0.0"
```
🤔 `RxGesture` is `1.0.0`.
But `.bind(onNext:` was introduced with `1.0.1`.

I guess you have some cleaning to perform because it seems that Carthage tries to build the wrong version. Then, try to use `RxGesture 1.0.1` again.

**Comment by bidadh** [comment]
oh sorry as I mentioned earlier I had to specify the version in Cartfile == 1.0.0

here is my Cartfile:
```
github "Alamofire/Alamofire"
github "ReactiveX/RxSwift"
github "RxSwiftCommunity/RxAlamofire"
github "RxSwiftCommunity/RxOptional"
github "RxSwiftCommunity/RxGesture" == 1.0.0
github "RxSwiftCommunity/Action"
github "RxSwiftCommunity/RxSegue"
github "RxSwiftCommunity/RxSwiftExt" == 2.3.0
github "RxSwiftCommunity/RxDataSources"
github "devxoul/RxKeyboard"
github "Swinject/Swinject"
github "Swinject/SwinjectStoryboard"
github "adamwaite/Validator"
github "jasonnam/Navajo-Swift"
github "Hearst-DD/ObjectMapper"
github "MengTo/Spring" "swift3"
```

**Comment by jegnux** [comment]
@bidadh Yes but can't you try again with `1.0.1`? Why doesn't it work with `1.0.1` exactly?

**Comment by bidadh** [comment]
I'm trying to find the reason

**Comment by bidadh** [comment]
I created an empty directory with a new Cartfile without any specific versions. It passes RxGesture with 1.0.1! and fails for RxSwiftExt

I then removed RxSwiftExt from my original project and re-run carthage update and still fails on RxGesture 1.0.1!

I just deleted Carthage directory and also removed RxSwiftExt from Cartfile in my original project and ran carthage update command again.  I'll update you with the result.

**Comment by bidadh** [comment]
Deleted Carthage directory and re-run `carthage update` works!
I still have an issue with RxSwiftExt and I have to specify 2.3.0 version but that's not relevant

all good now.

Thanks

**Comment by jegnux** [comment]
@bidadh ok great. I hope you'll find a solution for RxSwiftExt. Can I consider this issue resolved?

**Comment by bidadh** [comment]
yes. resolved.
Thanks @jegnux 

RxSwiftExt has been resolved in the new version(2.4.1)
